andela/hc-bigas-opus
# Ft user list priority #155965585

# What does this PR do?

* Adds the requirement that the job failure notification should be sent to the relevant team members as far as the scope of the failed job is concerned.

### Description of Task to be completed?

* Add is_alerted boolean field to the API models
* Add logic to is_alerted field in the sendalerts.py file to control to whom the alerts should be sent to in the team as far as the scope of the failed job is concerned

### How should this be manually tested?

* Execute `./manage.py runserver` then `./manage.py ensuretriggers` then `./manage.py sendalerts`. in separate terminals
* Login with a valid email address and confirm login from your Inbox.
* Go to account settings and in Team Access, set the team name and Invite a member
and then set priority.

### What are the relevant pivotal tracker stories?

[#155965585](https://www.pivotaltracker.com/story/show/155965585)
